lib/gollum/views/helpers.rb

```diff
@@ -2,6 +2,15 @@
 
 module Precious
   module Views
+    
+    module AppHelpers
+      def extract_page_dir(path)
+        return path if @page_dir.to_s.empty?
```

Comment:
Why convert `@page_dir` to a String here? If `@page_dir` is non-empty, it will evaluate as true, and if it's `nil`, it will evaluate to false. So this should also work:
```ruby
return path unless @page_dir
```